js/apps/system/_admin/aardvark/APP/react/src/views/g6graphs/AttributesInfo.js

```diff
@@ -0,0 +1,14 @@
+import { Tag } from 'antd';
```

Comment:
We have also created a new `<Tag />` component, but we are we not using that anywhere. Do we need to delete that, or use it here?